crates/compiler/can/src/expr.rs

```diff
@@ -1916,7 +1922,9 @@ fn canonicalize_var_lookup(
         // Look it up in the env!
         match env.qualified_lookup(scope, module_name, ident, region) {
             Ok(symbol) => {
-                output.references.insert_value_lookup(symbol);
+                output
+                    .references
+                    .insert_value_lookup(symbol, QualifiedReference::Qualified);
```

Comment:
I wonder if we should shove `output`/`references` into `env`, related to comment above about the duality of mutable inputs vs immutable outputs. Not actionable right now, but I'm pretty sure we should only be appending to `references` anyway, and presumably everytime we call `env.lookup` that corresponds 1-1 to a value/type reference lookup.

Reply:
This sounds like a good idea. I believe it would simplify a lot of code. If it's ok with you, I'd rather do that in a future PR to prevent the diff from getting even bigger.

Reply:
For sure, just noting.